curation/domain_services.py

```diff
@@ -0,0 +1,73 @@
+"""Provide lookup services for functional domains."""
+from pathlib import Path
+from datetime import datetime
+from curation import PROJECT_ROOT
+import logging
+import csv
+from ftplib import FTP
+from typing import List
+
+
+logger = logging.getLogger('fusion_backend')
+logger.setLevel(logging.DEBUG)
+
+
+class DomainService():
```

Comment:
Is it useful to add another field for `interpro ID` in DomainForm? Ex: user enters interpro id and both ID and domain name is in response?

Reply:
Good q -- I think that's out of scope right now, but could be something supported in the future.